api-service/node/services/hugepages.go

```diff
@@ -41,7 +41,7 @@ func (n *Node) SetHugepages(ctx context.Context, h *protos.Hugepages) (*protos.H
 	}
 
 	msg := []byte(strconv.Itoa(int(hugepages.Pages)))
-	err := ioutil.WriteFile(hugepagesPath, msg, 0644)
+	err := ioutil.WriteFile(hugepagesPath, msg, 0600)
```

Comment:
@asquare14 This change is not required as the the permission for the `nr_hugepages` file is `0644`

```console
akhil@MayaData:/sys/kernel/mm/hugepages/hugepages-2048kB $ ls -l
total 0
-r--r--r-- 1 root root 4096 May  7 12:55 free_hugepages
-rw-r--r-- 1 root root 4096 May  7 12:55 nr_hugepages
-rw-r--r-- 1 root root 4096 May  7 12:55 nr_hugepages_mempolicy
-rw-r--r-- 1 root root 4096 May  7 12:55 nr_overcommit_hugepages
-r--r--r-- 1 root root 4096 May  7 12:55 resv_hugepages
-r--r--r-- 1 root root 4096 May  7 12:55 surplus_hugepages
```

Reply:
Thanks. Closing it